202007_noise_car/compute_cl_noise_full.py

```diff
@@ -0,0 +1,49 @@
+import healpy as hp
+import mapsims
+from glob import glob
+import pickle
+import numpy as np
+import sys
+import pixell
+
+tube = sys.argv[1]
+m = {}
+cl = []
+folder = "output/noise/0000/"
+
+sim = mapsims.from_config(
+    ["common.toml", "noise.toml"], override={"channels": "tube:" + tube}
+)
+nside = mapsims.get_default_so_resolution(sim.channels[0])
+lmax = 3*nside-1
+noise = sim.other_components["noise"]
+inverse_noise_variance = noise.get_inverse_variance(tube)
+hitmap, _ = noise.get_hitmaps(tube)
+pixarea_map = noise.pixarea_map
+
+alm={}
+for i, ch in enumerate(mapsims.parse_channels("tube:" + tube)[0]):
+    hitmap[i] /= hitmap[i].max()
+
+    filename = glob(folder + f"*{ch.tag}*1_of_1*")[0]
+    print("reading " + filename)
+    m[i] = pixell.enmap.read_map(filename)
+    m[i][np.isnan(m[i])] = 0
+    
+    alm[i] = pixell.curvedsky.map2alm(
+        np.sqrt(hitmap[i])*m[i], lmax=lmax
```

Comment:
@msyriac I am trying to compare spectra, but I think I have a wrong normalization of the spectra, do I have to weight the pixels by the pixel area somehow?

![SAT_P(2)](https://user-images.githubusercontent.com/383090/87755953-dd5f2280-c7bc-11ea-83e2-42a95b3f5068.png)

The white noise line in the plots is just `wn = noise.get_white_noise_power(tube, sky_fraction)` where `sky_fraction = np.sum(mask*pixarea_map) /np.pi / 4.`

Reply:
@msyriac would you have time to take a look at these?